third_party/gpus/cuda_configure.bzl

```diff
@@ -374,6 +374,13 @@ def _lib_name(lib, cpu_value, version="", static=False):
       if version:
         version = ".%s" % version
       return "lib%s.so%s" % (lib, version)
+  elif cpu_value == "FreeBSD":
```

Comment:
This appears to be the same as the previous clause. Couldn't you say `cpu_value in ("Linux", "FreeBSD")`?